README.md

```diff
@@ -1,2 +1,87 @@
-# ega-fuse
-ega fuse client
+# EGA FUSE Client
+This is a Java Native Runtime (JNR) based FUSE client to access the EGA Data REST API. This client will allow access 
+to authorized EGA Archive files by presenting them in a vitual directory, where then can be used like regular files, 
+without first having to download them.
+
+## Prerequisite dependencies
+1. Maven
+2. Java
+
+#### Linux
+
+[`libfuse`](https://github.com/libfuse/libfuse) needs to be installed.
+
+#### Ubuntu
+```
+sudo apt-get install libfuse-dev
+``` 
```

Comment:
Should there be an example installation command for installing libfuse on Linux (similar to the MacOS block)? And why is there only a command for Ubuntu? Please make sure the Linux and Ubuntu commands match the format of the MacOS command.

Reply:
For Linux, we need to install `libfuse` & for macOS, we need to install  `osxfuse`
I have written both their installation commands in the doc